SportsHub/UnitTests/MockData/UserLoginDTOMockData.cs

```diff
@@ -0,0 +1,24 @@
+﻿using SportsHub.AppService.Authentication.Models.DTOs;
+
+namespace UnitTests.MockData;
+
+public class UserLoginDTOMockData
+{
+    public static UserLoginDTO GetUserWithUsername()
+    {
+        return new UserLoginDTO()
+        {
```

Comment:
for the future try to use Fixture where it is possible

Reply:
I will try to add it with the changes here